src/track/track.cpp

```diff
@@ -762,6 +795,10 @@ void Track::removeCue(const CuePointer& pCue) {
     QMutexLocker lock(&m_qMutex);
     disconnect(pCue.get(), 0, this, 0);
     m_cuePoints.removeOne(pCue);
+    if (pCue->getType() == Cue::LOAD) {
+        m_cuePoint = 0.0;
```

Comment:
These 2 lines of code might become some kind of _reset()_ operation on the new _CuePosition_ class.